pkg/reconciler/buildrun/buildrun.go

```diff
@@ -466,6 +466,17 @@ func (r *ReconcileBuildRun) Reconcile(ctx context.Context, request reconcile.Req
 				}
 			}
 
+			if buildRun.Status.Output != nil {
+				if len(buildRun.Status.Output.Vulnerabilities) > 0 {
+					if buildRun.Spec.BuildSpec.Output.VulnerabilityScan.FailOnFinding {
+						return reconcile.Result{}, resources.UpdateConditionWithFalseStatus(ctx, r.client, buildRun, fmt.Sprintf("Vulnerabilities have been found in the output image. For detailed information, see kubectl --namespace %s logs %s --container step-image-processing",
+							buildRun.Namespace,
+							lastTaskRun.Status.PodName,
+						), "VulnerabilitiesFound")
```

Comment:
In the message, should we say "check the BuildRun status, or see kubectl ..." ?

Can we put `VulnerabilitiesFound` into a constant ? Would be nearby here: https://github.com/shipwright-io/build/blob/6ee1cd84cd8252c2e2dd42df14a5227d007f7d6b/pkg/apis/build/v1beta1/buildrun_types.go#L108

Reply:
Done